app/src/components/Modals/activity-modal/activity-modal-body.tsx

```diff
@@ -540,12 +544,14 @@ const ActivityModalBody = ({
                     }
                     color="content.tertiary"
                     h="48px"
+                    fontSize="14px"
                     shadow="1dp"
                     pos="relative"
                     zIndex={10}
                     {...register("activity.ch4EmissionFactorUnit")}
                   >
-                    {""}
+                    CH4/{""}
+                    {/*  TODO input the right units*/}
```

Comment:
You can get the units from the manual input hierarchy file. Please also create a Jira ticket with highest priority for this.